bin/gftools-push-status.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Check push status of families on Dev, Sandbox and Production servers.
+
+This script will determine whether the families listed in
+to_production.txt and to_sandbox.txt in the google/fonts dir have been
+pushed to their respective servers within an allocated date.
```

Comment:
"within an allocated date" is an odd turn of phrase

Reply:
Ok, I'll use "from a specific date".